packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -7,6 +7,8 @@ const managerRoot = {
 const URI = {
   billing: '#/billing/history',
   contacts: '#/contacts/services',
+  domain: '#/configuration/domain/:domain',
```

Comment:
Those are already handled by the `service` 2API and accessible in `/hub/billingServices` payload. I think we should avoid duplicating those as it will be a pain to maintain if we have them in different places